README.md

```diff
@@ -1006,7 +1006,7 @@ resources. You can download it via `ilab model download`.
 If you have a server running, stop the server you have running by entering `ctrl+c` keys in the terminal running the server.
 
 *🍎 Mac only:* Before serving the newly trained model you must convert it to work with
-the `ilab` cli. The `ilab model convert` command converts the new model into quantized [GGUF](https://medium.com/@sandyeep70/ggml-to-gguf-a-leap-in-language-model-file-formats-cd5d3a6058f9) format which is required by the server to host the model in the `ilab model serve` command.
+the `ilab` core. The `ilab model convert` command converts the new model into quantized [GGUF](https://medium.com/@sandyeep70/ggml-to-gguf-a-leap-in-language-model-file-formats-cd5d3a6058f9) format which is required by the server to host the model in the `ilab model serve` command.
```

Comment:
```suggestion
the InstructLab core. The `ilab model convert` command converts the new model into quantized [GGUF](https://medium.com/@sandyeep70/ggml-to-gguf-a-leap-in-language-model-file-formats-cd5d3a6058f9) format which is required by the server to host the model in the `ilab model serve` command.
```

Reply:
Capitalize "Core" I think and add "package"

Reply:
updated to `InstructLab Core package`